Example_MeliCardDrawer/Example_MeliCardDrawer/Controllers/ViewController.swift

```diff
@@ -182,3 +187,46 @@ extension ViewController {
         }
     }
 }
+
+// MARK: Toggle SafeArea feature
+extension ViewController {
+    @IBAction func didChangeSafeArea(_ sender: UISwitch) {
+        if(sender.isOn) {
```

Comment:
### Suggestion

Remove () from this condition verification to standardize code like you did on line 69